test/include/seqan3/test/pretty_printing.hpp

```diff
@@ -20,8 +20,8 @@ namespace seqan3
 //!\cond DEV
 //!\brief Overload for the googletest PrintTo function that always delegates to our debug_stream.
 template <typename t>
-    requires true // tricks the compiler to consider this as more specialized than googletests generic PrintTo
-void PrintTo (t const & v, std::ostream * out)
+    requires requires (t && v) { debug_stream << v; }
```

Comment:
I think, if you want to test that both l-value and r-values can be put to the debug_stream you'll need to change  `debug_stream << std::forward<t>(v);` as well.

Reply:
I just wanted to capture `v` by `&&` so I don't add a const qualifier before passing it to the debug_stream. Since within PrinTo `v` is always an l-value I do not need to test for debug-streaming r-values do I?

Reply:
I don't know the signature of PrintTo from gtest though

Reply:
The signature from gtest has const & in it... so you might be right. I'll check this tomorrow morning